app/forms/user_piv_cac_verification_form.rb

```diff
@@ -0,0 +1,69 @@
+class UserPivCacVerificationForm
+  include ActiveModel::Model
+
+  attr_accessor :x509_dn_uuid, :x509_dn, :token, :error_type, :nonce, :user
+
+  validates :token, presence: true
+  validates :nonce, presence: true
+  validates :user, presence: true
+
+  def submit
+    success = valid? && valid_token?
+
+    FormResponse.new(success: success, errors: {})
```

Comment:
A reminder to populate this errors hash with the appropriate error in subsequent PRs.

Reply:
Definitely. I'm going to be looking at making a super-class for the two different piv/cac forms that will handle that logic since it'll need to transform the errors from the identity-pki service into ActiveModel terms.